api/src/main/resources/application.properties

```diff
@@ -1,3 +1,7 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/foroAlura
 spring.datasource.username=root
 spring.datasource.password=Yineth103010
+
+spring.jpa.hibernate.ddl-auto=create
```

Comment:
We don't want the property `spring.jpa.hibernate.ddl-auto=create` since it is tied to what modifies the database on the fly, please remove.